src/probe_matcher.cpp

```diff
@@ -479,7 +479,7 @@ void ProbeMatcher::list_probes(ast::Program* prog)
         std::cout << probe_type << ":" << match_print << std::endl;
         if (bt_verbose) {
           for (auto& param : param_lists[match])
-            std::cout << "    " << param << std::endl;
+            LOG(V1) << "    " << param;
```

Comment:
We could possibly drop the above `if (bt_verbose)` (although then the loop will always be executed).

Reply:
> although then the loop will always be executed

Yeah that's why I left it in.